bing_bert/deepspeed_bsz16K_lamb_config.json

```diff
@@ -1,6 +1,6 @@
 {
   "train_batch_size": 16384,
-  "train_micro_batch_size_per_gpu": 64,
+  "train_micro_batch_size_per_gpu": 16,
   "steps_per_print": 1000,
   "prescale_gradients": true,
   "optimizer": {
```

Comment:
Hi @jeffra, I believe this and other json files need an update to use the `gradient_clipping` parameter instead of the deprecated `max_grad_norm` parameter? I get the following error:
```
/usr/local/lib/python3.6/dist-packages/deepspeed/pt/deepspeed_light.py:470: UserWarning: 'max_grad_norm' is not supported as an optimizer parameter, please switch to using the deepspeed parameter 'gradient_clipping' see: https://www.deepspeed.ai/docs/config-json/#gradient-clipping for more details
```